lte/gateway/configs/mme.yml

```diff
@@ -27,3 +27,8 @@ enable_apn_correction: false
 apn_correction_map_list:
         - imsi_prefix: "00101"
           apn_override: "magma.ipv4"
+s1ap_zmq_th: 2000000 # microseconds (delay threshold used for dropping initial UE message)
```

Comment:
nit: might be good to append all the configs with `_us` to have the unit in the name itself?